### PR TITLE
feat: various changes

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**
I've combined all my PRs into this one.

- Updates the embed sent to Discord
	- Author field now has a hyperlink to https://github.com/sponsors/IanMitchell and an icon
	- Footer field now has the Sentry project name
- Reverts #35 due to issues
- Uses Imgbot to optimise images
- Bumps the `JasonEtco/is-sponsor-label-action` workflow dependency to `v2`

### CODE REVIEW

File: lib/message.ts [hunk 1]
```diff
@@ -15,8 +15,9 @@ export default function createMessage(event) {
 
   embed.setAuthor({
     name: "Sentry → Discord",
-    url: "https://sentrydiscord.dev",
```
Comment (IanMitchell) [inline lib/message.ts:18]: I think I'd rather keep this in the footer than move it to the header!

**Comment by WilliamDavidHarrison** [comment]
@IanMitchell I have reverted the sponsor part to the footer.

File: pages/index.tsx [hunk 2]
```diff
@@ -84,12 +84,13 @@ export default function Home({ events, webhooks }) {
             </p>
             <div className="mt-5 max-w-md mx-auto sm:flex sm:justify-center md:mt-8">
               <div className="rounded-md shadow">
-                <Link
-                  href="/create"
-                  onClick={onStartClick}
-                  className="w-full flex items-center justify-center px-8 py-3 border border-transparent text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 md:py-4 md:text-lg md:px-10"
-                >
-                  Get started
+                <Link href="/create">
```
Comment (IanMitchell) [inline pages/index.tsx:87]: Think we need to revert this change as well since the new Next.js doesn't allow nested `<a>` but then we are looking good!

**Comment by WilliamDavidHarrison** [comment]
@IanMitchell Done 😄

**Comment by IanMitchell** [comment]
thanks 🙏 😄

**Comment by WilliamDavidHarrison** [comment]
No problem! I like helping out!
